### ISSUE TITLE
[Feat] Add full support for Ventura

### ISSUE THREAD

**Issue by Jeehut** [FlineDev/ReMafoX#9]
## Problem Statement

Currently the app seems not to be optimized for macOS Ventura. The project browser has some borders that look weird and in some places there might be UI glitches. Also the window management is sub-par.

## Suggested Solution

Fully test the app on Ventura and optimize it for it, including a reimplementation of the window management system using SwiftUI 4.

## Additional Considerations

None.

**Comment by Jeehut** [comment]
@Jeehut Make sure to check also that this bug reported for Ventura is fixed, too:
https://github.com/FlineDev/ReMafoX/issues/37

**Comment by Jeehut** [comment]
I have just fixed multiple UI glitches and #37 while testing support for Ventura. It seems, all the issues I've seen during the Beta period except for #37 have been worked out by the Apple teams, I can't find other issues.

But the improvements to the window management I had planned for the Ventura update are not possible. I had mistakenly thought that the new [openWindow](https://developer.apple.com/documentation/swiftui/environmentvalues/openwindow) API would also provide a way to close a window properly using SwiftUI and do other stuff related to window management. But this seems not to be the case, it's only here for opening new windows. There was never a problem with opening new windows in the app though, so I can't make many improvements as part of a Ventura update on window management.

Closing this one – it'll be part of the October update `1.1.0`.
